cvs diff: Diffing sw
cvs diff: Diffing sw/inc
cvs diff: Diffing sw/inc/pch
cvs diff: Diffing sw/prj
Index: sw/prj/build.lst
===================================================================
RCS file: /cvs/sw/sw/prj/build.lst,v
retrieving revision 1.22
retrieving revision 1.22.362.1
diff -p -u -u -p -b -w -B -r1.22 -r1.22.362.1
--- sw/prj/build.lst	27 Jun 2007 13:15:21 -0000	1.22
+++ sw/prj/build.lst	28 Mar 2008 11:22:01 -0000	1.22.362.1
@@ -1,6 +1,7 @@
 sw	sw	:	connectivity OOo:writerperfect svx stoc uui NULL
 sw	sw										usr1	-	all	sw_mkout NULL
 sw	sw\inc									nmake		-	all	sw_inc NULL
+sw	sw\uiconfig\layout									nmake	-	all	sw_layout NULL
 sw	sw\prj									get		-	all	sw_prj NULL
 sw	sw\source\ui\inc						nmake	-	all	sw_uinc NULL
 sw	sw\source\core\inc						get		-	all	sw_cinc NULL
Index: sw/prj/d.lst
===================================================================
RCS file: /cvs/sw/sw/prj/d.lst,v
retrieving revision 1.22
retrieving revision 1.22.440.2
diff -p -u -u -p -b -w -B -r1.22 -r1.22.440.2
--- sw/prj/d.lst	16 Apr 2007 15:39:57 -0000	1.22
+++ sw/prj/d.lst	31 Mar 2008 14:50:42 -0000	1.22.440.2
@@ -107,3 +107,5 @@ mkdir: %_DEST%\xml%_EXT%\uiconfig\module
 ..\uiconfig\sglobal\statusbar\*.xml %_DEST%\xml%_EXT%\uiconfig\modules\sglobal\statusbar\*.xml
 ..\uiconfig\swform\statusbar\*.xml %_DEST%\xml%_EXT%\uiconfig\modules\swform\statusbar\*.xml
 ..\uiconfig\swreport\statusbar\*.xml %_DEST%\xml%_EXT%\uiconfig\modules\swreport\statusbar\*.xml
+
+..\%__SRC%\bin\*-layout.zip %_DEST%\pck%_EXT%\*.*
cvs diff: Diffing sw/qa
cvs diff: Diffing sw/qa/complex
cvs diff: Diffing sw/qa/complex/accessibility
cvs diff: Diffing sw/qa/complex/checkColor
cvs diff: Diffing sw/qa/complex/indeterminateState
cvs diff: Diffing sw/qa/complex/writer
cvs diff: Diffing sw/qa/complex/writer/testdocuments
cvs diff: Diffing sw/qa/core
cvs diff: Diffing sw/qa/unoapi
cvs diff: Diffing sw/qa/unoapi/testdocuments
cvs diff: Diffing sw/sdi
cvs diff: Diffing sw/source
cvs diff: Diffing sw/source/core
cvs diff: Diffing sw/source/core/SwNumberTree
cvs diff: Diffing sw/source/core/access
cvs diff: Diffing sw/source/core/attr
cvs diff: Diffing sw/source/core/bastyp
cvs diff: Diffing sw/source/core/crsr
cvs diff: Diffing sw/source/core/doc
cvs diff: Diffing sw/source/core/docnode
cvs diff: Diffing sw/source/core/draw
cvs diff: Diffing sw/source/core/edit
cvs diff: Diffing sw/source/core/except
cvs diff: Diffing sw/source/core/fields
cvs diff: Diffing sw/source/core/frmedt
cvs diff: Diffing sw/source/core/graphic
cvs diff: Diffing sw/source/core/inc
cvs diff: Diffing sw/source/core/layout
cvs diff: Diffing sw/source/core/objectpositioning
cvs diff: Diffing sw/source/core/ole
cvs diff: Diffing sw/source/core/para
cvs diff: Diffing sw/source/core/sw3io
cvs diff: Diffing sw/source/core/swg
cvs diff: Diffing sw/source/core/table
cvs diff: Diffing sw/source/core/text
cvs diff: Diffing sw/source/core/tox
cvs diff: Diffing sw/source/core/txtnode
cvs diff: Diffing sw/source/core/undo
cvs diff: Diffing sw/source/core/unocore
cvs diff: Diffing sw/source/core/view
cvs diff: Diffing sw/source/filter
cvs diff: Diffing sw/source/filter/ascii
cvs diff: Diffing sw/source/filter/basflt
cvs diff: Diffing sw/source/filter/html
cvs diff: Diffing sw/source/filter/inc
cvs diff: Diffing sw/source/filter/rtf
cvs diff: Diffing sw/source/filter/writer
cvs diff: Diffing sw/source/filter/ww1
cvs diff: Diffing sw/source/filter/ww8
cvs diff: Diffing sw/source/filter/ww8/dump
cvs diff: Diffing sw/source/filter/xml
cvs diff: Diffing sw/source/ui
cvs diff: Diffing sw/source/ui/app
cvs diff: Diffing sw/source/ui/cctrl
cvs diff: Diffing sw/source/ui/chrdlg
cvs diff: Diffing sw/source/ui/config
cvs diff: Diffing sw/source/ui/dbui
cvs diff: Diffing sw/source/ui/dialog
cvs diff: Diffing sw/source/ui/dochdl
cvs diff: Diffing sw/source/ui/docvw
cvs diff: Diffing sw/source/ui/envelp
cvs diff: Diffing sw/source/ui/fldui
cvs diff: Diffing sw/source/ui/fmtui
cvs diff: Diffing sw/source/ui/frmdlg
cvs diff: Diffing sw/source/ui/globdoc
cvs diff: Diffing sw/source/ui/inc
cvs diff: Diffing sw/source/ui/index
cvs diff: Diffing sw/source/ui/lingu
cvs diff: Diffing sw/source/ui/misc
cvs diff: Diffing sw/source/ui/ribbar
cvs diff: Diffing sw/source/ui/shells
cvs diff: Diffing sw/source/ui/smartmenu
cvs diff: Diffing sw/source/ui/table
cvs diff: Diffing sw/source/ui/uiview
cvs diff: Diffing sw/source/ui/uno
cvs diff: Diffing sw/source/ui/utlui
cvs diff: Diffing sw/source/ui/web
cvs diff: Diffing sw/source/ui/wrtsh
cvs diff: Diffing sw/uiconfig
cvs diff: Diffing sw/uiconfig/layout
Index: sw/uiconfig/layout/delzip
===================================================================
RCS file: sw/uiconfig/layout/delzip
diff -N sw/uiconfig/layout/delzip
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ sw/uiconfig/layout/delzip	3 Apr 2008 08:38:35 -0000	1.1.2.2
@@ -0,0 +1 @@
+empty
\ No newline at end of file
Index: sw/uiconfig/layout/layout.mk
===================================================================
RCS file: sw/uiconfig/layout/layout.mk
diff -N sw/uiconfig/layout/layout.mk
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ sw/uiconfig/layout/layout.mk	31 Mar 2008 15:01:30 -0000	1.1.2.1
@@ -0,0 +1,33 @@
+# TODO: move to solenv/inc
+# copies: sw/uiconfig/layout svx/uiconfig/layout
+
+TRALAY=tralay
+XML_DEST=$(DLLDEST)
+XML_LANGS=$(alllangiso)
+
+ALL_XMLS=$(foreach,i,$(XML_FILES) $(XML_DEST)/$i) $(foreach,i,$(XML_LANGS) $(foreach,j,$(XML_FILES) $(XML_DEST)/$i/$j))
+
+# Must remove the -j (no duplicate base file names) flag
+ZIPUPDATE=-u
+XML_ZIP = $(PRJNAME)-layout
+
+ALLTAR: $(XML_ZIP)
+
+$(XML_ZIP): $(ALL_XMLS)
+
+ZIP1DIR=$(XML_DEST)
+ZIP1TARGET=$(XML_ZIP)
+ZIP1LIST=$(ALL_XMLS:s@$(XML_DEST)/@@)
+
+$(foreach,i,$(XML_LANGS) $(XML_DEST)/$i/%.xml): %.xml
+	-$(MKDIR) $(@:d)
+	@echo $(foreach,i,$(XML_LANGS) $(XML_DEST)/$i/%.xml): %.xml
+	$(TRALAY) -m localize.sdf -o $(XML_DEST) -l $(XML_LANGS:f:t" -l ") $<
+
+$(XML_DEST)/%.xml: %.xml
+	-$(MKDIR) $(@:d)
+	$(COPY) $< $@
+
+# Don't want to overwrite filled localize.sdf with empty template
+template.sdf:
+	$(foreach,i,$(XML_FILES) $(TRALAY) -l en-US $i) > $@
Index: sw/uiconfig/layout/localize.sdf
===================================================================
RCS file: sw/uiconfig/layout/localize.sdf
diff -N sw/uiconfig/layout/localize.sdf
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ sw/uiconfig/layout/localize.sdf	28 Mar 2008 11:22:01 -0000	1.1.2.1
@@ -0,0 +1,559 @@
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	en-US	Word Count				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	en-US	Current selection				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	en-US	Words:				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FI_CURRENTWORD_label				0	en-US	0				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	en-US	Characters:				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FI_CURRENTCHARACTER_label				0	en-US	00				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	en-US	Whole document				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	en-US	Words:				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FI_DOCWORD_label				0	en-US	000				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	en-US	Characters:				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FI_DOCCHARACTER_label				0	en-US	0000				20080227 16:28:34
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	af	Huidige seleksie				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ar	التحديد الحالي				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	as-IN	বর্তমানৰ বাছনী				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	be-BY	Пазначанае				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	bg	Текущ избор				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	br	Diuz oberiat				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	bs	Trenutni izbor				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ca	Selecció actual				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	cs	Současný výběr				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	cy	Dewis presennol				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	da	Aktuel markering				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	de	Aktuelle Auswahl				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	dz	ད་ལྟོའི་སེལ་འཐུ།				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	el	Τρέχουσα επιλογή				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	en-GB	Current selection				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	en-ZA	Current selection				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	eo	Aktuala elekto				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	es	Selección actual				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	et	Aktiivne valik				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	fa	انتخاب فعلی				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	fi	Nykyinen valinta				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	fr	Sélection active				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ga	Roghnúchán reatha				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	gl	Selección actual				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	gu	હાલની પસંદગી				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	gu-IN	હાલની પસંદગી				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	he	הבחירה הנוכחית				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	hi-IN	मौजूदा चयन				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	hr	Trenutni odabir				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	hu	Aktuális kijelölés				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	it	Selezione				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ja	現在の選択				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ka	მიმდინარე მონიშვნა				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	km	ការ​ជ្រើស​បច្ចុប្បន្ន				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ko	현재 선택 사항				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ku	Hilbijartina derbazbar				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	lt	Šiuo ~metu pažymėta sritis				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	mk	Тековен избор				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ml-IN	നിലവിലുള്ള തെരഞ്ഞെടുപ്പ്				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	mr-IN	वर्तमान निवड				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	nb	Gjeldende utvalg				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ne	हालको चयन				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	nl	Huidige selectie				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	nn	Gjeldande utval				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	nr	Ikhetho lanje				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ns	Kgetho ya gona bjale				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	or-IN	ଚଳିତ ମନୋନଯନ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	pa-IN	ਮੌਜੂਦਾ ਚੋਣ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	pl	Zaznaczenie				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	pt	Selecção actual				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	pt-BR	Seleção atual				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ru	Текущее выделение				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	rw	ihitamo rigezweho				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	sh	Tekući odeljak				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	sk	Súčasný výber				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	sl	Trenutna izbira				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	sr	Текући одељак				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ss	Kukhetsa kwanyalo				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	st	Kgetho ya jwale				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	sv	Aktuell markering				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	sw-TZ	Current selection				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ta	நடப்பு தேர்த்தெடுத்தல்				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ta-IN	நடப்பு தேர்த்தெடுத்தல்				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	te-IN	ప్రస్ధుత ఎంపిక				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	tg	Интихоби ҷорӣ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	th	การเลือกปัจจุบัน				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ti-ER	ህሉው ምርጫ 				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	tr	Geçerli seçim				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ts	Ku hlawula ka sweswi				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	uk	Поточний вибір				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ur-IN	حالیہ انتخاب 				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	uz	Joriy tanlov				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	ve	Khetho ino				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	vi	Vùng chọn hiện tái				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	xh	Ukukhetha kwangoku				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	zh-CN	当前选定内容				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	zh-TW	目前的選取				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_CURRENT_text				0	zu	Lokho okukhethile manje				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	af	Hele dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ar	المستند بالكامل				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	as-IN	সম্পূৰ্ণ ডকুমেন্ট				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	be-BY	Дакумент цалкам				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	bg	Целият документ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	br	Teul a-bezh				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	bs	Cijeli dokumenat				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ca	Tot el document				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	cs	Celý dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	cy	Dogfen gyfan				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	da	Hele dokumentet				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	de	Gesamtes Dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	dz	ཡིག་ཆ་ཧྲིལ་བུམ།				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	el	Ολόκληρο το έγγραφο				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	en-GB	Whole document				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	en-ZA	Whole document				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	eo	Tuta dokumento				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	es	Documento completo				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	et	Terve dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	fa	سراسر نوشتار				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	fi	Koko asiakirja				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	fr	Document entier				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ga	An cháipéis iomlán				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	gl	Documento completo				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	gu	પુરૂ દસ્તાવેજ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	gu-IN	પુરૂ દસ્તાવેજ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	he	כל המסמך				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	hi-IN	पूर्ण दस्तावेज				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	hr	Cijeli dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	hu	Teljes dokumentum				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	it	Intero documento				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ja	ドキュメント全体				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ka	სრული დოკუმენტი				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	km	ឯកសារ​ទាំងមូល				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ko	전체 문서				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ku	Li hemû belgeyê				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	lt	Visas dokumentas				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	mk	Цел документ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ml-IN	മുഴുവന് ഡോക്കുമെന്റും				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	mr-IN	संपूर्ण दस्तऍवज				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	nb	Hele dokumentet				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ne	संपूर्ण कागजात				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	nl	Heel document				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	nn	Heile dokumentet				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	nr	Idokhumente yoke				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ns	Tokumente ka moka				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	or-IN	ସଂପୂର୍ଣ୍ଣ ଦଲିଲ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	pa-IN	ਪੂਰਾ ਦਸਤਾਵੇਜ਼				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	pl	Cały dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	pt	Todo o documento				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	pt-BR	Documento inteiro				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ru	Весь документ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	rw	Inyandiko yose				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	sh	Ceo dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	sk	Celý dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	sl	Celoten dokument				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	sr	Цео документ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ss	Idokhumenti yonkhe				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	st	Tokomane kaofela				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	sv	Hela dokumentet				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	sw-TZ	Whole document				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ta	முழு ஆவணம்				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ta-IN	முழு ஆவணம்				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	te-IN	పత్రమంత				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	tg	Тамоми ҳуҷҷат				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	th	ทั้งเอกสาร				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ti-ER	ምሉእ ሰነድ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	tr	Tüm belge				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ts	Dokumente hinkwayo				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	uk	Весь документ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ur-IN	پوری دستاویز				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	uz	Butun hujjat				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	ve	Ḽiṅwalo ḽoṱhe				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	vi	Toàn tài liệu				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	xh	Uxwebhu luphelele				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	zh-CN	整篇文档				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	zh-TW	整份文件				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FL_DOC_text				0	zu	Lonke ifayela				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	af	Karakters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ar	محارف:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	as-IN	আখৰবোৰ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	be-BY	Знакі:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	bg	Знаци:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	br	Arouezennoù :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	bs	Znakovi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ca	Caràcters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	cs	Znaky:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	cy	Nodau:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	da	Tegn:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	de	Zeichen:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	dz	ཡིག་འབྲུ་ཚུ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	el	Χαρακτήρες:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	en-GB	Characters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	en-ZA	Characters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	eo	Signoj:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	es	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	et	Märke:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	fa	نویسه‌ها:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	fi	Merkit:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	fr	Caractères :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ga	Carachtair:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	gl	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	gu	અક્ષરો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	gu-IN	અક્ષરો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	he	תווים:‏				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	hi-IN	संप्रतीक:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	hr	Znakova:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	hu	Karakterek száma:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	it	Caratteri:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ja	文字数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ka	ასოები:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	km	តួអក្សរ ៖				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ko	문자:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ku	Tîp:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	lt	Rašmenys:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	mk	Знаци:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ml-IN	അക്ഷരങ്ങള്				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	mr-IN	अक्षरे:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	nb	Tegn:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ne	क्यारेक्टरहरू:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	nl	Tekens:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	nn	Teikn:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	nr	Amakharektha:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ns	Ditlhaka:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	or-IN	ଅକ୍ଷରଗୁଡିକ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	pa-IN	ਅੱਖਰ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	pl	Znaki:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	pt	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	pt-BR	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ru	Символы:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	rw	Inyuguti:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	sh	Znakova:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	sk	Znaky:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	sl	Znaki:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	sr	Знакова:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ss	Tinhlavu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	st	Ditlhaku:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	sv	Tecken:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	sw-TZ	Viwambo:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ta	எழுத்துக்கள்:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ta-IN	எழுத்துக்கள்:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	te-IN	అక్షరములు:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	tg	Рамзҳо:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	th	อักขระ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ti-ER	ፊደላት				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	tn	Didiragatsi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	tr	Karakter:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ts	Swivumbeko:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	uk	Символи:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ur-IN	الفاظ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	uz	Belgilar:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	ve	Zwiteṅwa:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	vi	Ký tự :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	xh	Imibhalo enjengoonobumba:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	zh-CN	字符数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	zh-TW	字元：				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTCHARACTER_label				0	zu	Izimpawu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	af	Woorde:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ar	كلمات:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	as-IN	শব্দবোৰ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	be-BY	Словы:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	bg	Думи:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	br	Gerioù :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	bs	Riječi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ca	Paraules:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	cs	Slova:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	cy	Geiriau:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	da	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	de	Wörter:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	dz	མིང་ཚིག་ཚུ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	el	Λέξεις:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	en-GB	Words:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	en-ZA	Words:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	eo	Vortoj:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	es	Palabras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	et	Sõnu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	fa	کلمات:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	fi	Sanat:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	fr	Mots :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ga	Focail:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	gl	Palabras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	gu	શબ્દો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	gu-IN	શબ્દો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	he	מילים				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	hi-IN	शब्द:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	hr	Riječi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	hu	Szavak száma:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	it	Parole:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ja	単語数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ka	სიტყვები:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	km	ពាក្យ ៖				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ko	단어:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ku	Peyv:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	lt	Žodžiai				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	mk	Зборови:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ml-IN	വാക്കുകള്				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	mr-IN	शब्द:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	nb	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ne	शब्दहरू:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	nl	Woorden:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	nn	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	nr	Amagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ns	Mantšu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	or-IN	ଶବ୍ଦଗୁଡିକ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	pa-IN	ਸ਼ਬਦ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	pl	Wyrazy:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	pt	Palavras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	pt-BR	Palavras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ru	Слова:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	rw	Amagambo:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	sh	Reči:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	sk	Slová:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	sl	Število besed:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	sr	Речи:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ss	Emagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	st	Mantswe:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	sv	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	sw-TZ	Words:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ta	சொற்கள்:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ta-IN	சொற்கள்:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	te-IN	పదాలు:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	tg	Калимаҳо:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	th	คำ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ti-ER	ቃላት				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	tr	Kelime:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ts	Marito:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	uk	Слова				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ur-IN	الفاظ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	uz	Soʻzlar:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	ve	Maipfi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	vi	Từ :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	xh	Amagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	zh-CN	字数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	zh-TW	文字：				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_CURRENTWORD_label				0	zu	Amagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	af	Karakters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ar	محارف:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	as-IN	আখৰবোৰ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	be-BY	Знакі:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	bg	Знаци:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	br	Arouezennoù :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	bs	Znakovi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ca	Caràcters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	cs	Znaky:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	cy	Nodau:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	da	Tegn:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	de	Zeichen:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	dz	ཡིག་འབྲུ་ཚུ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	el	Χαρακτήρες:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	en-GB	Characters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	en-ZA	Characters:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	eo	Signoj:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	es	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	et	Märke:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	fa	نویسه‌ها:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	fi	Merkit:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	fr	Caractères :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ga	Carachtair:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	gl	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	gu	અક્ષરો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	gu-IN	અક્ષરો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	he	תווים:‏				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	hi-IN	संप्रतीक:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	hr	Znakova:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	hu	Karakterek száma:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	it	Caratteri:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ja	文字数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ka	ასოები:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	km	តួអក្សរ ៖				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ko	문자:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ku	Tîp:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	lt	Rašmenys:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	mk	Знаци:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ml-IN	അക്ഷരങ്ങള്				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	mr-IN	अक्षरे:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	nb	Tegn:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ne	क्यारेक्टरहरू:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	nl	Tekens:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	nn	Teikn:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	nr	Amakharektha:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ns	Ditlhaka:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	or-IN	ଅକ୍ଷରଗୁଡିକ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	pa-IN	ਅੱਖਰ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	pl	Znaki:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	pt	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	pt-BR	Caracteres:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ru	Символы:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	rw	Inyuguti:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	sh	Znakova:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	sk	Znaky:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	sl	Znaki:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	sr	Знакова:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ss	Tinhlavu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	st	Ditlhaku:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	sv	Tecken:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	sw-TZ	Viwambo:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ta	எழுத்துக்கள்:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ta-IN	எழுத்துக்கள்:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	te-IN	అక్షరములు:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	tg	Рамзҳо:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	th	อักขระ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ti-ER	ፊደላት				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	tn	Didiragatsi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	tr	Karakter:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ts	Swivumbeko:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	uk	Символи:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ur-IN	الفاظ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	uz	Belgilar:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	ve	Zwiteṅwa:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	vi	Ký tự :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	xh	Imibhalo enjengoonobumba:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	zh-CN	字符数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	zh-TW	字元：				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCCHARACTER_label				0	zu	Izimpawu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	af	Woorde:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ar	كلمات:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	as-IN	শব্দবোৰ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	be-BY	Словы:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	bg	Думи:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	br	Gerioù :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	bs	Riječi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ca	Paraules:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	cs	Slova:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	cy	Geiriau:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	da	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	de	Wörter:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	dz	མིང་ཚིག་ཚུ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	el	Λέξεις:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	en-GB	Words:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	en-ZA	Words:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	eo	Vortoj:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	es	Palabras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	et	Sõnu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	fa	کلمات:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	fi	Sanat:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	fr	Mots :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ga	Focail:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	gl	Palabras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	gu	શબ્દો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	gu-IN	શબ્દો:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	he	מילים				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	hi-IN	शब्द:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	hr	Riječi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	hu	Szavak száma:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	it	Parole:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ja	単語数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ka	სიტყვები:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	km	ពាក្យ ៖				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ko	단어:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ku	Peyv:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	lt	Žodžiai				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	mk	Зборови:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ml-IN	വാക്കുകള്				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	mr-IN	शब्द:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	nb	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ne	शब्दहरू:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	nl	Woorden:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	nn	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	nr	Amagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ns	Mantšu:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	or-IN	ଶବ୍ଦଗୁଡିକ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	pa-IN	ਸ਼ਬਦ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	pl	Wyrazy:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	pt	Palavras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	pt-BR	Palavras:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ru	Слова:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	rw	Amagambo:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	sh	Reči:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	sk	Slová:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	sl	Število besed:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	sr	Речи:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ss	Emagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	st	Mantswe:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	sv	Ord:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	sw-TZ	Words:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ta	சொற்கள்				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ta-IN	சொற்கள்				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	te-IN	పదాలు:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	tg	Калимаҳо:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	th	คำ:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ti-ER	ቃላት				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	tr	Kelime:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ts	Marito:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	uk	Слова				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ur-IN	الفاظ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	uz	Soʻzlar:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	ve	Maipfi:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	vi	Từ :				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	xh	Amagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	zh-CN	字数:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	zh-TW	文字：				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	FT_DOCWORD_label				0	zu	Amagama:				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	af	Woordtelling				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ar	عدد الكلمات				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	as-IN	শব্দ গণনা 				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	be-BY	Колькасць словаў				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	bg	Брой думи				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	br	Stadegoù				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	bs	Broj riječi				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ca	Recompte de paraules				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	cs	Počet slov				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	cy	Cyfrif Geiriau				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	da	Ordoptælling				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	de	Wortanzahl				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	dz	མིང་ཚིག་གྱངས་ཁ་རྐྱབས།				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	el	Αρίθμηση λέξεων				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	en-GB	Word Count				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	en-ZA	Word Count				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	eo	Vortnombro				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	es	Contar palabras				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	et	Sõnade loendamine				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	fa	تعداد کلمات				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	fi	Sanamäärä				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	fr	Statistiques				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ga	Líon na bhFocal				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	gl	Contaxe de palabras!				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	gu	શબ્દની ગણતરી				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	gu-IN	શબ્દની ગણતરી				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	he	סה‏"כ מילים				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	hi-IN	शब्द गणना				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	hr	Brojanje riječi				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	hu	Szavak száma				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	it	Conteggio parole				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ja	ワードカウント				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ka	სიტყვათა რაოდენობა				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	km	ចំនួន​ពាក្យ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ko	단어 개수				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ku	Hijmartina Peyvan				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	lt	Statistika				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	mk	Броење зборови				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ml-IN	വാക്കുകളുടെ എണ്ണം				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	mr-IN	शब्द गणना				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	nb	Ordtelling				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ne	शब्द गणना				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	nl	Woorden tellen				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	nn	Ordteljing				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	nr	Bala Amagama				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ns	Palo ya Mantšu				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	or-IN	ଶବ୍ଦ ଗଣନା				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	pa-IN	ਸ਼ਬਦ ਗਿਣਤੀ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	pl	Licznik słów				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	pt	Contar palavras				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	pt-BR	Contagem de palavras				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ru	Количество слов				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	rw	Kubara amagambo				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	sh	Broj reči				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	sk	Počet slov				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	sl	Štetje besed				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	sr	Број речи				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ss	Kubalwa kwemagama				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	st	Palo ya Mantswe				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	sv	Räkna ord				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	sw-TZ	Word Count				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ta	சொல் எண்ணிக்கை				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ta-IN	சொல் எண்ணிக்கை				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	te-IN	పదాల లెక్క				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	tg	Шумораи калимаҳо				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	th	นับคำ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ti-ER	ምቑጻር ቃላት				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	tr	Kelime Sayısı				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ts	Ku hlayela marito				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	uk	Кількість слів				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ur-IN	  الفاظ شمار				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	uz	Soʻzlar soni				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	ve	Mbalo ya Maipfi				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	vi	Tổng số từ				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	xh	Ubalo Lwamagama				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	zh-CN	字数统计				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	zh-TW	字數				2002-02-02 02:02:02
+layout	sw\uiconfig\layout\wordcount.xml	0	layout	DLG_WORDCOUNT_title				0	zu	Inani Lamagama				2002-02-02 02:02:02
Index: sw/uiconfig/layout/makefile.mk
===================================================================
RCS file: sw/uiconfig/layout/makefile.mk
diff -N sw/uiconfig/layout/makefile.mk
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ sw/uiconfig/layout/makefile.mk	31 Mar 2008 15:01:30 -0000	1.1.2.4
@@ -0,0 +1,54 @@
+#*************************************************************************
+#
+#   OpenOffice.org - a multi-platform office productivity suite
+#
+#   $RCSfile$
+#
+#   $Revision$
+#
+#   last change: $Author$ $Date$
+#
+#   The Contents of this file are made available subject to
+#   the terms of GNU Lesser General Public License Version 2.1.
+#
+#
+#     GNU Lesser General Public License Version 2.1
+#     =============================================
+#     Copyright 2005 by Sun Microsystems, Inc.
+#     901 San Antonio Road, Palo Alto, CA 94303, USA
+#
+#     This library is free software; you can redistribute it and/or
+#     modify it under the terms of the GNU Lesser General Public
+#     License version 2.1, as published by the Free Software Foundation.
+#
+#     This library is distributed in the hope that it will be useful,
+#     but WITHOUT ANY WARRANTY; without even the implied warranty of
+#     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#     Lesser General Public License for more details.
+#
+#     You should have received a copy of the GNU Lesser General Public
+#     License along with this library; if not, write to the Free Software
+#     Foundation, Inc., 51 Franklin Street, 5th Floor, Boston,
+#     MA  02110-1301  USA
+#
+#*************************************************************************
+
+PRJ=../..
+PRJNAME=sw
+TARGET=layout
+
+.INCLUDE : settings.mk
+
+.IF "$(ENABLE_LAYOUT)" == "TRUE"
+
+all: ALLTAR
+
+XML_FILES=wordcount.xml
+
+.INCLUDE : layout.mk
+
+.ELSE # ENABLE_LAYOUT != TRUE
+all .PHONY:
+.ENDIF # ENABLE_LAYOUT != TRUE
+
+.INCLUDE :  target.mk
Index: sw/uiconfig/layout/wordcount.xml
===================================================================
RCS file: sw/uiconfig/layout/wordcount.xml
diff -N sw/uiconfig/layout/wordcount.xml
--- /dev/null	1 Jan 1970 00:00:00 -0000
+++ sw/uiconfig/layout/wordcount.xml	4 Apr 2008 09:14:08 -0000	1.1.2.2
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- This is a template.  i18n translation is not performed in-place;
+     i18n translated xml files are generated from this template by
+     transex3/layout/tralay.  !-->
+
+<modaldialog xmlns="http://openoffice.org/2007/layout"
+             xmlns:cnt="http://openoffice.org/2007/layout/container"
+             help-id="HID_DLG_WORDCOUNT"
+             id="DLG_WORDCOUNT"
+             sizeable="true" moveable="true" optimumsize="true"
+             has_border="true" sv-look="true" _title="Word Count">
+    <vbox border="5" spacing="5">
+        <fixedline id="FL_CURRENT" _text="Current selection"/>
+	<hbox>
+	    <vbox cnt:padding="13" spacing="5">
+		<hbox>
+		    <fixedtext id="FT_CURRENTWORD" _label="Words:"/>
+		    <flow cnt:padding="60"/>
+		    <fixedtext id="FI_CURRENTWORD" right="true" _label="0"/>
+		</hbox>
+		<hbox>
+		    <fixedtext id="FT_CURRENTCHARACTER" _label="Characters:"/>
+		    <flow cnt:padding="60"/>
+		    <fixedtext id="FI_CURRENTCHARACTER" right="true" _label="00"/>
+		</hbox>
+	    </vbox>
+	</hbox>
+	<!-- if we could do cnt:cnt:padding="-13" here, we would not have to close
+	     and reope the hbox/vbox thing !-->
+        <fixedline id="FL_DOC" _text="Whole document"/>
+	<hbox>
+	    <vbox cnt:padding="13" spacing="5">
+		<hbox>
+		    <fixedtext id="FT_DOCWORD" _label="Words:"/>
+		    <fixedtext id="FI_DOCWORD" right="true" _label="000"/>
+		</hbox>
+		<hbox>
+		    <fixedtext id="FT_DOCCHARACTER" _label="Characters:"/>
+		    <fixedtext id="FI_DOCCHARACTER" right="true" _label="0000"/>
+		</hbox>
+	    </vbox>
+	</hbox>
+        <fixedline cnt:padding="1" id="FL_BOTTOM"/>
+	<dialogbuttonhbox spacing="5">
+            <flow/>
+            <okbutton id="PB_OK"/>
+            <helpbutton id="PB_HELP"/>
+        </dialogbuttonhbox>
+    </vbox>
+</modaldialog>
cvs diff: Diffing sw/uiconfig/sglobal
cvs diff: Diffing sw/uiconfig/sglobal/accelerator
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/de
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/en-US
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/es
cvs diff: Diffing sw/uiconfig/sglobal/accelerator/fr
cvs diff: Diffing sw/uiconfig/sglobal/menubar
cvs diff: Diffing sw/uiconfig/sglobal/statusbar
cvs diff: Diffing sw/uiconfig/sglobal/toolbar
cvs diff: Diffing sw/uiconfig/sweb
cvs diff: Diffing sw/uiconfig/sweb/accelerator
cvs diff: Diffing sw/uiconfig/sweb/accelerator/de
cvs diff: Diffing sw/uiconfig/sweb/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/sweb/accelerator/en-US
cvs diff: Diffing sw/uiconfig/sweb/accelerator/es
cvs diff: Diffing sw/uiconfig/sweb/accelerator/fr
cvs diff: Diffing sw/uiconfig/sweb/menubar
cvs diff: Diffing sw/uiconfig/sweb/statusbar
cvs diff: Diffing sw/uiconfig/sweb/toolbar
cvs diff: Diffing sw/uiconfig/swform
cvs diff: Diffing sw/uiconfig/swform/accelerator
cvs diff: Diffing sw/uiconfig/swform/accelerator/de
cvs diff: Diffing sw/uiconfig/swform/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swform/accelerator/es
cvs diff: Diffing sw/uiconfig/swform/accelerator/fr
cvs diff: Diffing sw/uiconfig/swform/menubar
cvs diff: Diffing sw/uiconfig/swform/statusbar
cvs diff: Diffing sw/uiconfig/swform/toolbar
cvs diff: Diffing sw/uiconfig/swreport
cvs diff: Diffing sw/uiconfig/swreport/accelerator
cvs diff: Diffing sw/uiconfig/swreport/accelerator/de
cvs diff: Diffing sw/uiconfig/swreport/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/swreport/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swreport/accelerator/es
cvs diff: Diffing sw/uiconfig/swreport/accelerator/fr
cvs diff: Diffing sw/uiconfig/swreport/menubar
cvs diff: Diffing sw/uiconfig/swreport/statusbar
cvs diff: Diffing sw/uiconfig/swreport/toolbar
cvs diff: Diffing sw/uiconfig/swriter
cvs diff: Diffing sw/uiconfig/swriter/accelerator
cvs diff: Diffing sw/uiconfig/swriter/accelerator/de
cvs diff: Diffing sw/uiconfig/swriter/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/swriter/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swriter/accelerator/es
cvs diff: Diffing sw/uiconfig/swriter/accelerator/fr
cvs diff: Diffing sw/uiconfig/swriter/menubar
cvs diff: Diffing sw/uiconfig/swriter/statusbar
cvs diff: Diffing sw/uiconfig/swriter/toolbar
cvs diff: Diffing sw/uiconfig/swxform
cvs diff: Diffing sw/uiconfig/swxform/accelerator
cvs diff: Diffing sw/uiconfig/swxform/accelerator/de
cvs diff: Diffing sw/uiconfig/swxform/accelerator/en-GB
cvs diff: Diffing sw/uiconfig/swxform/accelerator/en-US
cvs diff: Diffing sw/uiconfig/swxform/accelerator/es
cvs diff: Diffing sw/uiconfig/swxform/accelerator/fr
cvs diff: Diffing sw/uiconfig/swxform/menubar
cvs diff: Diffing sw/uiconfig/swxform/statusbar
cvs diff: Diffing sw/uiconfig/swxform/toolbar
cvs diff: Diffing sw/util
Index: sw/util/makefile.mk
===================================================================
RCS file: /cvs/sw/sw/util/makefile.mk,v
retrieving revision 1.66
retrieving revision 1.66.18.2
diff -p -u -u -p -b -w -B -r1.66 -r1.66.18.2
--- sw/util/makefile.mk	12 Mar 2008 13:00:47 -0000	1.66
+++ sw/util/makefile.mk	28 Mar 2008 10:18:04 -0000	1.66.18.2
@@ -199,11 +199,6 @@ SHL3STDLIBS= \
             $(SALLIB) \
 			$(SOTLIB)
 
-.IF "$(ENABLE_LAYOUT)" == "TRUE"
-LINKFLAGS+= -L../$(PRJ)/layout/$(INPATH)/lib
-SHL3STDLIBS += -ltklayout$(UPD)$(DLLPOSTFIX)
-.ENDIF # ENABLE_LAYOUT == TRUE
-
 SHL3LIBS=   $(SLB)$/swui.lib
 LIB3TARGET = $(SLB)$/swui.lib
 
cvs diff: Diffing sw/util/defs
cvs diff: Diffing sw/xml
